graphein/protein_ligand/edges/intramolecular.py

```diff
@@ -0,0 +1,325 @@
+"""Featurization functions for graph edges."""
+# Graphein
+# Author: Arian Jamasb <arian@jamasb.io>, Eric Ma, Charlie Harris
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+from __future__ import annotations
+
+import logging
+import os
+import subprocess
+from pathlib import Path
+from typing import Optional
+
+import networkx as nx
+import pandas as pd
+
+from graphein.protein.utils import download_pdb
+
+log = logging.getLogger(__name__)
+
+
+def peptide_bonds(G: nx.Graph) -> nx.Graph:
```

Comment:
Probably better to avoid duplication if there are no changes to the underlying protein graph functions